admin/src/types.ts

```diff
@@ -82,7 +81,7 @@ export enum PROPOSAL_STATUS {
 // NOTE: sync with backend/grant/utils/enums.py ProposalStage
 export enum PROPOSAL_STAGE {
   PREVIEW = 'PREVIEW',
-  FUNDING_REQUIRED = 'FUNDING_REQUIRED',
+  FUNDING_REQUIRED = 'FUNDING_REQUIRED', // kept for backwards compatibility 
```

Comment:
Scripts should be updated to remove FUNDING_REQUIRED, and this stage can be removed.